verchk.go

```diff
@@ -107,10 +88,24 @@ func (v *versionChecker) writeClientUUID() (string, error) {
 	if err != nil {
 		return "", err
 	}
+	//this just checks to see if there was anything at all in the file.
```

Comment:
comments should have a space between the `//` and the comment text, because the compiler uses `//foo` type stuff for some compiler directives.